install.ps1

```diff
@@ -241,6 +242,21 @@ if (-not $noChecks.IsPresent) {
     }
 }
 
+# Check Internet connectivity and return boolean value, exit if value is 'false'
```

Comment:
The internet check needs to be inside the `if (-not $noChecks.IsPresent)`.

Reply:
Done